sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,133 @@
+<!-- eslint-disable vue/valid-v-slot -->
+<template>
+  <v-card class="mb-3">
+    <v-container>
+      <p class="label-bold-normal mb-0 text-body-2">
+        This feature can be used to ignore specific restrictions for specific
+        applications. Restrictions can be ignored for one, or all, disbursements
+        associated with an application. If a restriction is ignored, it's effect
+        will be suspended. For example, if a restriction typically prevents an
+        eCert from getting produced and sent to NSLSC 5 days prior to study
+        start dates, then ignoring that restriction will enable the eCert to get
+        produced and sent to NSLSC, thereby 'bypassing' the restriction. Please
+        see training material for additional information.
+      </p>
+    </v-container>
+  </v-card>
+  <v-card>
+    <v-container>
```

Comment:
Please use the existing layout `body-header-container`

```vue
<body-header-container :enable-card-view="true">
    <template #header
      ><body-header title="History of Bypassed Restrictions"
    /></template>
    <content-group class="mt-4">
      <toggle-content
        :toggled="!bypassedRestrictions.bypasses.length || isDraftApplication"
        :message="getToggleContentMessage()"
      >
        <v-data-table
          :headers="ApplicationRestrictionManagementHeaders"
          :items="bypassedRestrictions.bypasses"
          :items-per-page="DEFAULT_PAGE_LIMIT"
        >
          <template #[`item.restrictionType`]="{ item }">
            {{ item.restrictionType }}
          </template>
          <template #[`item.restrictionCode`]="{ item }">
            {{ item.restrictionCode }}
          </template>
          <template #[`item.isActive`]="{ item }">
            <status-chip-bypass :is-active="item.isActive" />
          </template>
          <template #[`item.id`]>
            <v-btn
              color="primary"
              variant="text"
              class="text-decoration-underline"
              prepend-icon="fa:far fa-file-alt"
            >
              View Details</v-btn
            >
          </template>
          <template #[`item.isBypassActive`]="{ item }">
            <v-btn
              :color="getRemoveBypassColor(item.isBypassActive)"
              :disabled="!item.isBypassActive"
            >
              {{ getRemoveBypassLabel(item.isBypassActive) }}</v-btn
            >
          </template>
        </v-data-table>
      </toggle-content>
    </content-group>
  </body-header-container>
```

Reply:
Updated.